### PR TITLE
43 create sidebar component

### PR DESCRIPTION
**Pull request by Vadis00**
using:
`
  <app-sidebar><p>Hello, world!</p></app-sidebar>`

open sidebar:
`  constructor(private sideBarService: SideBarService) {}
  open() {
    this.sideBarService.toggle();
  }
`

### CODE REVIEW

**Review by ismv — CHANGES_REQUESTED** [review]
Conflicts Resolving

**Review by Its-OP — APPROVED** [review]
Please, use better naming in your git flow. 'bugs fixed', 'finished', etc. are neither complete nor informative titles.

**Review by Its-OP — CHANGES_REQUESTED** [review]
And fix merge conflicts

File: frontend/src/app/app.module.ts [hunk 1]
```diff
@@ -24,9 +24,11 @@ import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
     ToastrModule.forRoot(),
     FontAwesomeModule,
   ],
+
   providers: [
-    BrowserAnimationsModule,
+    BrowserAnimationsModule
```
Comment (ihor-il) [inline frontend/src/app/app.module.ts]: Do you use Prettier?

Comment (Vadis00) [reply frontend/src/app/app.module.ts]: Yes, I already set it to work "Format in save mode"

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by ismv — APPROVED** [review]

**Pull request merged**